src/software/ai/hl/stp/tactic/passer/passer_tactic.cpp

```diff
@@ -0,0 +1,55 @@
+#include "software/ai/hl/stp/tactic/passer/passer_tactic.h"
+
+#include "shared/constants.h"
+#include "software/ai/hl/stp/action/intercept_ball_action.h"
+#include "software/ai/hl/stp/action/kick_action.h"
+#include "software/ai/hl/stp/action/move_action.h"
+#include "software/ai/hl/stp/action/stop_action.h"
+#include "software/logger/logger.h"
+#include "software/world/ball.h"
+
+PasserTactic::PasserTactic(Pass pass)
+    : Tactic(true, {RobotCapability::Kick, RobotCapability::Move}),
```

Comment:
this should really be loop_forever = false although it doesn't matter

Reply:
changed

Reply:
changed